assets/assets.go

```diff
@@ -18,6 +19,14 @@ import (
 	"github.com/ipfs/interface-go-ipfs-core/path"
 )
 
+//go:embed init-doc dir-index-html/dir-index.html dir-index-html/knownIcons.txt
+var dir embed.FS
+
+var (
+	// AssetHash a cryptographic hash of all embedded assets
```

Comment:
Actually this need be updated.
```suggestion
	// AssetHash a non-cryptographic hash of all embedded assets
```

Reply:
Nice catch - missed it.